ubuntu_image/tests/test_builder.py

```diff
@@ -768,6 +771,152 @@ def test_populate_filesystems_bogus_type(self):
             self.assertEqual(
                 str(cm.exception), 'Invalid part filesystem type: 801')
 
+    def test_populate_filesystems_lk_bootloader(self):
+        # We do a bit-wise copy when the file system has no type.
```

Comment:
This comment should describe the actual test that's being performed. I know it's been copy-pasted a few times before, but let's try making it better and getting back to actually documenting the test cases better.